### PR TITLE
Updates for 1.8.4 release

### PR DESCRIPTION
**Pull request by smarthi**

### CODE REVIEW

**Comment by tteofili** [comment]
+1

File: src/main/jbake/content/news/release-184.ad [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  = Apache OpenNLP 1.8.4 released
+  Apache OpenNLP
+  2017-12-24
+  :jbake-type: post
+  :jbake-tags: community
+  :jbake-status: published
+  :category: news
+  :idprefix:
+
+  The Apache OpenNLP team is pleased to announce the release of Apache OpenNLP 1.8.4.
```
Comment (kinow) [inline src/main/jbake/content/news/release-184.ad]: Didn't see how it renders, but it appears to have been duplicated by accident? The license header appears twice, as well as these first paragraphs...

**Review by jzonthemtn — APPROVED** [review]
Make sure the edits I made are ok.

**Review by kinow — APPROVED** [review]

**Comment by smarthi** [comment]
+1 to merge

**Comment by kinow** [comment]
+1

**Pull request merged**